pimd/pim_ssm.h

```diff
@@ -19,7 +19,12 @@
 #ifndef PIM_SSM_H
 #define PIM_SSM_H
 
+#if PIM_IPV == 4
 #define PIM_SSM_STANDARD_RANGE "232.0.0.0/8"
+#else
+#define PIM_SSM_STANDARD_RANGE "FF3x::/32"
```

Comment:
Is this only just for "printing subnet"? Or will be used in inet_ntop() in the future?